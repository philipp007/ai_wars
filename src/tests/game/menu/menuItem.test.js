import React from 'react';
import MenuItem from '../../../game/menu/menuItem';
import renderer from 'react-test-renderer';

test('MenuItem should be rendered', () => { 
    const menu = renderer.create(
        <MenuItem key='1' name="Test" selected={ false } />
    );

    let tree = menu.toJSON();
    expect(tree).toMatchSnapshot();
});