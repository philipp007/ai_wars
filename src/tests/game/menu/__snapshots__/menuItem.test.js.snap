// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem should be rendered 1`] = `
<div
  className="menuItem"
>
  <span>
    Test
  </span>
</div>
`;
